lib/preneeds/middleware/response/clean_response.rb

```diff
@@ -4,7 +4,7 @@ module Middleware
     module Response
       class CleanResponse < Faraday::Response::Middleware
```

Comment:
We probably also want an html error response middleware - based on pinging endpoints to get connection info, there's likelihood of a HTML error response if something is really fubared (like a 404).